tutorial/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+cmake_minimum_required(VERSION 3.21)
+
+project(
+  musica-demo
+  VERSION 0.1
+  LANGUAGES CXX C Fortran
+)
+
+# mkdir build
+# cd build
+# cmake -DMUSICA_INSTALL_DIR=<musica_install_dir> ..
+
+set(MUSICA_INCLUDE_DIR "${MUSICA_INSTALL_DIR}/include")
+set(MUSICA_LIB_DIR "${MUSICA_INSTALL_DIR}/lib")
+
+message(STATUS "${MUSICA_INCLUDE_DIR}")
+message(STATUS "${MUSICA_LIB_DIR}")
+
+add_executable(demo_c demo.c)
+add_executable(demo_f demo.f90)
+
+target_include_directories(demo_c PUBLIC ${MUSICA_INCLUDE_DIR})
+target_link_directories(demo_c PUBLIC ${MUSICA_LIB_DIR})
+
+target_include_directories(demo_f PUBLIC ${MUSICA_INCLUDE_DIR})
+target_link_directories(demo_f PUBLIC ${MUSICA_LIB_DIR})
+target_link_libraries(demo_f musica-fortran musica stdc++)
```

Comment:
Instead of using `target_link_directories`, could you use `target_link_libraries` as it is discouraged to use as stated in the docs? I also wonder whether you need them. When you just use `target_link_librararies` to link musica, does it fail?

Reply:
yes, I would add any tutorial code to the test suite

Reply:
Great, @dwfncar, could you make `demo_*` as a test?

Reply:
Without the `target_link_libraries` I get the link errors `/usr/bin/ld: cannot fine -lmusica-fortran` (and -`lmusica`)

Reply:
Added demo.f90 as a unit test in fortran/test/unit/CMakeLists.txt, will add demo.c when it has actual content.

Reply:
hmm maybe instead of`tutorial/` being the top level directory it might need to be a subdirectory of`fortran/` like `micm`? Then it could link to `musica-fortran` library. What do you think?

Reply:
Yeah, I would recommend the tutorial tests being in `micm/fortran/test/tutorial` and you should be able to just use the `create_standard_fortran_test()` macro (I think). That macro links tests against the `musica-fortran` library. Similar to this: https://github.com/NCAR/musica/blob/69bcb2efed80afab88be74809a44df43717b9a73/fortran/test/unit/CMakeLists.txt#L6-L8